lib/clients/backbeat-2017-07-01.api.json

```diff
@@ -994,7 +1048,7 @@
         "GetObjectMetadata": {
             "http": {
                 "method": "GET",
-                "requestUri": "/_/metadata/getobject/{Bucket}/{Key+}"
+                "requestUri": "/_/metadata/default/bucket/{Bucket}/{Key+}"
```

Comment:
Rahul did a great refactor here: https://github.com/scality/cloudserver/blob/development/8.1/lib/routes/routeMetadata.js
Since I haven't fully tested it yet, my current api changes are based off: https://github.com/scality/cloudserver/commit/d09b6da7beb0c9f0fa5d8b8b8fbbc9235c525de3

However, based on my understanding,  `attributes` would be specifying we want the information of the bucket itself, whereas specifying `bucket` would just mean we're accessing a `bucket` for information on the contents of the bucket.

Reply:
As it's already implemented in cloudserver, it's probably a minor thing at this point: I just would have expected:

Route to retrieve bucket metadata:
`/_/metadata/default/attributes/{Bucket}`

Route to retrieve object metadata:
`/_/metadata/default/attributes/{Bucket}/{Key+}`